package.json

```diff
@@ -27,77 +27,77 @@
+    "@wdio/spec-reporter": "5.15.2",
+    "@wdio/sync": "5.16.0",
+    "c8": "4.1.5",
+    "chai": "4.2.0",
+    "chromedriver": "77.0.0",
+    "concurrently": "4.1.2",
+    "cross-env": "5.2.1",
+    "eslint": "6.6.0",
+    "eslint-config-google": "0.14.0",
+    "eslint-plugin-header": "3.0.0",
+    "eslint-plugin-import": "2.18.2",
     "express": "4.17.1",
-    "glob": "^7.1.4",
+    "glob": "7.1.5",
     "grammkit": "0.7.0",
-    "js-green-licenses": "^1.1.0",
-    "minimist": "^1.2.0",
-    "mocha": "^5.2.0",
-    "mocha-chrome": "^2.0.0",
-    "morgan": "^1.9.1",
-    "node-webcrypto-ossl": "^1.0.39",
-    "npm-run-all": "^4.1.5",
-    "pegjs": "^0.10.0",
-    "rollup": "^1.17.0",
-    "rollup-plugin-commonjs": "^9.3.4",
-    "rollup-plugin-ignore": "^1.0.5",
-    "rollup-plugin-multi-entry": "^2.1.0",
-    "rollup-plugin-node-resolve": "^4.2.4",
-    "rollup-plugin-typescript2": "^0.21.2",
-    "sloc": "^0.2.1",
-    "ts-pegjs": "^0.2.6",
-    "tslint": "^5.20.0",
-    "type-coverage": "^2.0.1",
-    "typedoc": "^0.15.0",
-    "wdio-chromedriver-service": "^5.0.2",
-    "webdriverio": "^5.11.1",
-    "webpack": "^4.32.2",
-    "webpack-cli": "^3.2.3"
+    "js-green-licenses": "1.1.0",
+    "minimist": "1.2.0",
+    "mocha": "5.2.0",
+    "mocha-chrome": "2.2.0",
+    "morgan": "1.9.1",
+    "node-webcrypto-ossl": "1.0.48",
+    "npm-run-all": "4.1.5",
+    "pegjs": "0.10.0",
+    "rollup": "1.26.3",
+    "rollup-plugin-commonjs": "9.3.4",
+    "rollup-plugin-ignore": "1.0.5",
+    "rollup-plugin-multi-entry": "2.1.0",
+    "rollup-plugin-node-resolve": "4.2.4",
+    "rollup-plugin-typescript2": "0.21.2",
+    "sloc": "0.2.1",
+    "ts-pegjs": "0.2.6",
+    "tslint": "5.20.1",
+    "type-coverage": "2.3.0",
+    "typedoc": "0.15.0",
+    "wdio-chromedriver-service": "5.0.2",
+    "webdriverio": "5.16.0",
+    "webpack": "4.41.2",
+    "webpack-cli": "3.3.10"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^1.2.9",
-    "assert": "^1.4.1",
-    "circular-dependency-plugin": "^5.2.0",
-    "esm": "^3.0.84",
-    "firebase": "^6.0.4",
-    "idb": "^2.1.3",
-    "jsrsasign": "^8.0.12",
-    "jsrsasign-util": "^1.0.0",
-    "jszip": "^3.2.1",
-    "node-fetch": "^2.3.0",
-    "pouchdb": "^7.1.1",
-    "pouchdb-adapter-memory": "^7.1.1",
-    "pouchdb-debug": "^7.1.1",
-    "source-map-support": "^0.5.12",
-    "sourcemapped-stacktrace": "^1.1.9",
+    "@tensorflow/tfjs": "1.3.1",
+    "assert": "1.5.0",
+    "circular-dependency-plugin": "5.2.0",
+    "esm": "3.2.25",
+    "firebase": "6.6.2",
+    "idb": "2.1.3",
+    "jsrsasign": "8.0.12",
+    "jsrsasign-util": "1.0.0",
+    "jszip": "3.2.2",
+    "node-fetch": "2.6.0",
+    "pouchdb": "7.1.1",
+    "pouchdb-adapter-memory": "7.1.1",
+    "pouchdb-debug": "7.1.1",
+    "source-map-support": "0.5.16",
+    "sourcemapped-stacktrace": "1.1.11",
     "typescript": "3.7.2",
-    "ws": "~6.0.0"
+    "ws": "6.0.0"
   },
   "repository": {
     "type": "git",
```

Comment:
Let's fix the versions of our `engines`, too: 
```
"engines": {
    "node": ">= 10.13.0",	 
    "npm": ">= 6.9.0"	 
  }	  
```
We use an .nvmrc to enforce a fixed `node` version. We should do the same or similar for npm.

Reply:
Perhaps in a separate PR so we can make sure this goes smoothly before we fiddle that, but defer to @shans thought.